plugins/scaffolder-backend/src/service/router.ts

```diff
@@ -796,6 +799,22 @@ export async function createRouter(
         name: step.name ?? step.action,
       }));
 
+      const dryRunId = uuid();
+      const contentsPath = resolveSafeChildPath(
+        workingDirectory,
+        `dry-run-content-${dryRunId}`,
+      );
+
+      const templateInfo = {
+        entityRef: 'template:default/dry-run',
```

Comment:
Could we grab the entity ref from the passed in `template` here?

Reply:
I may be missing something really basic. I was under the impression there was no entityRef here because we were in dry run. I thought it was passed to us in the body in the real post task event here. 

```
 const templateRef: string = req.body.templateRef;
```

But this wasn't available in the dry run execution so we assign it this one?

Reply:
The template is provided as input to the dry-run that we can call `stringifyEntityRef` on here: https://github.com/backstage/backstage/blob/151baa260d5e7b05ef369ebc2e626da85aaa41e0/plugins/scaffolder-backend/src/service/router.ts#L770

Reply:
I went ahead and made requested change. Sorry I am still a little new to TypeScript and I didn't realize that template type satisfied the parameters to `stringifyEntityRef`.

Just confirming we don't mind that this changes the value that it was previously right?